components/src/core/components/PopOverButton/PopOverButton.vue

```diff
@@ -51,11 +64,14 @@ export default defineComponent({
     'oxd-button': OxdButton,
     'oxd-pop-over-button-drop-down-option': PopOverButtonDropDownOption,
   },
-  emits: ['click', 'select'],
+  emits: ['click', 'update:modelValue'],
```

Comment:
this change will cause other places that using this component to change?

Reply:
this is checked Chamara, only the click event is used so far